pkg/api/object.go

```diff
@@ -72,6 +72,11 @@ func (s *Server) objectAPIEntries() []*Entry {
 			Method:  "DELETE",
 			Handler: s.deleteObject,
 		},
+		{
+			Path:    ObjectPrefix,
```

Comment:
this could implement the feature. however, deleting all objects could be a disastrous operation, and some people use `curl` to send the command, this `Path` increases the possibility of their faulty operation.